template/package-scripts.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Windows: Please do not use trailing comma as windows will fail with token error
+ */
+module.exports = {
+  scripts: {
+    default: 'webpack',
+    webpack: {
+      default: 'webpack.dev',
```

Comment:
Why do we have a Webpack command. Isn't this handled in the vue cli?

Reply:
Since this is from official vue cli webpack template, i don't thinks so ;)